### PR TITLE
Core user type What's New post

### PR DESCRIPTION
**Pull request by jkinmonth**
This needs to go live Jan 12, 10 am PST to match with the Core user in-product experience.

Will update with the correct Nerdbyte link by 1 pm PST, Jan 11.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by urbiz-nr** [comment]
@zuluecho9 - can you have a look at this tomorrow? Thanks!

File: src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: 'Introducing our new core user type
+' 
+summary: 'Unlock telemetry data access for even more developers in your org
+' 
+releaseDate: '2022-01-12' 
+learnMoreLink: 'https://example.com'
```
Comment (daynaslord) [inline src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md]: @jkinmonth to add the blog link here

File: src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+title: 'Introducing our new core user type
+' 
+summary: 'Unlock telemetry data access for even more developers in your org
+' 
+releaseDate: '2022-01-12' 
+learnMoreLink: 'https://example.com' 
+getStartedLink: 'https://example.com'
```
Comment (daynaslord) [inline src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md]: @jkinmonth to add a link to the NR1 UI or to the instructions in docs

File: src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+---
+
+Observability is better together. Our new core user type enables more developers than ever to understand code performance better, collaborate and prioritize work directly from their IDE, and access relevant error details including stack traces, logs, alerts, and more.
+
+**New core users enable developers to:** 
+* Optimize code performance and feature planning with access to telemetry data from production and pre-production environments directly in their IDE via the [New Relic CodeStream integration.](https://newrelic.com/codestream) 
+
```
Comment (daynaslord) [inline src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md]: Remove space here

File: src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+* Collaborate with DevOps, site reliability engineers (SREs), and ops engineers with a direct path from [error tracking](https://newrelic.com/platform/errors-inbox) in New Relic One to the relevant code block directly in their IDE.
+* Detect patterns and outliers in [log data](https://one.newrelic.com/launcher/logger.log-launcher?platform[accountId]=3386328&platform[timeRange][duration]=1800000&platform[$isFallbackTimeRange]=true&state=18f8ab72-f351-81d7-8b9d-d0a44b5b94a3) to resolve issues and improve code performance. 
+* Build and run [custom applications](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/build-new-relic-one/build-custom-new-relic-one-application/) in New Relic One to surface performance insights, optimize cloud costs, and more. 
+Onboard new developers with observability best practices and shared baselines for metrics.
+
+
+## Preview Period: Try it core user capabilities today
```
Comment (daynaslord) [inline src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md]: Change to --> Preview Period: Try core user capabilities today

File: src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+All existing basic users currently have free access to core user capabilities through Feb. 28, 2022, as part of a preview period. After the preview period, only full platform or core users can access the New Relic CodeStream integration, errors inbox, New Relic One custom apps, and advanced log management features.
```
Comment (daynaslord) [inline src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md:19]: Capitalize Errors Inbox?

File: src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+**To start viewing error tracking and other telemetry data directly within your IDE,** follow the [instructions](https://newrelic.com/codestream) to set up the New Relic * CodeStream integration.
+
+**To enable errors inbox** at your account, follow these steps. Afterwards, errors groups will appear in your inbox.
+From [one.newrelic.com](one.newrelic.com), select errors inbox from the top nav.
+
+* If this is your first time accessing [errors inbox](https://one.newrelic.com/launcher/errors-inbox.launcher?platform[accountId]=3386328&platform[timeRange][duration]=86400000&platform[$isFallbackTimeRange]=true&state=49e6d628-79ac-7022-1689-a6a6b7673f54), you will be prompted to select a workload in the top left.
+* If you have no workloads set up, you will be prompted to create one before you can use errors inbox.
+
```
Comment (daynaslord) [inline src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md:28]: Change to --> If you have no workloads set up, you will be prompted to [create one](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/workloads/use-workloads/) before you can use errors inbox.

Add --> Once you select your workload, your inbox should populate with error groups.

File: src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+## Explore core user capabilities in this walkthrough 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/04JP0ky_hjI" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Unlock core users at your account
+To add core users to your account, visit the plan management UI within New Relic One and enable core users with just a couple of clicks. Note that only users with a Billing user role in the Admin group can visit the plan management section of New Relic One.
```
Comment (daynaslord) [inline src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md]: To add core users to your account, head to the plan management UI within New Relic One by clicking the drop down menu and then 'Manage your plan.' Enable core users with just a few clicks. **Note:** Only users with a Billing user role in the Admin group can visit the plan management section of New Relic One.

**Review by daynaslord — APPROVED** [review]

**Comment by daynaslord** [comment]
@zuluecho9 @jkinmonth - Added some comments above

**Review by zuluecho9 — APPROVED** [review]

File: src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md (hunk 4 above)
Comment (jkinmonth) [inline src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md]: @zuluecho9 I found a typo in line 22, "it" should be removed.

New copy:

## Preview period: Try core user capabilities today. `

File: src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md (hunk 7 above)
Comment (jkinmonth) [inline src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md]: Made some adjustments to line 39 for accuracy. New copy below:

To start adding core users to your account, visit the plan management UI within New Relic One and enable core users by accepting the new terms. Note that some accounts may need to talk with their account team to unlock core users. In addition, only users with a Billing user role in the Admin group can visit the plan management section of New Relic One.

**Pull request merged**